src/storage-preview/azext_storage_preview/_transformers.py

```diff
@@ -122,5 +124,51 @@ def transform_storage_list_output(result):
     return list(result)
 
 
+# pylint: disable=unused-argument
 def transform_file_upload(result):
     return None
+
+
+def transform_queue_stats_output(result):
+    if isinstance(result, dict):
```

Comment:
Recommend to use `todict` to transform the reuslt to dict first. And your transform code could be applied to result no matter if it is a dictionary or not. https://github.com/microsoft/knack/blob/40d06da2c07c44afe285b467585a6da437d6bd21/knack/util.py#L122-L145

Reply:
As this func is recursively called in line 138, `isinstance(result, dict)` is to determine whether it's the end level or not.